src/solvers/refinement/string_refinement.cpp

```diff
@@ -665,10 +927,15 @@ void string_refinementt::add_lemma(
 /// \par parameters: an expression representing an array and an expression
 /// representing an integer
 /// \return an array expression or an array_of_exprt
-exprt string_refinementt::get_array(const exprt &arr, const exprt &size) const
+static exprt get_array(
```

Comment:
`super_get` and `max_string_length` can be `const`.

Reply:
Arguments have been made const.